interface.go

```diff
@@ -175,7 +175,7 @@ type LoopInRequest struct {
 	// call.
 	MaxMinerFee btcutil.Amount
 
-	// HtlcConfTarget specifies the targeted confirmation target for the
+	// HtlcConfirmations specifies the targeted confirmation target for the
```

Comment:
field name not actually changed

Reply:
also, the comment is a bit weird, maybe use something similar to the one for loop out below?